packages/client/src/helpers/client/formatError.ts

```diff
@@ -73,14 +74,18 @@ export const adaptError = (error: {
     return error;
   }
 
+  const extraParameters: Record<string, unknown> = {};
+  if (error.code) extraParameters['transportLayerErrorCode'] = error.code;
```

Comment:
I think this style is better, as it is easier to read and less prone to errors: 
```suggestion
  if (error.code) { 
    extraParameters['transportLayerErrorCode'] = error.code; 
  }
```

What do you think?

Reply:
Fixed!